org.freefilesync.FreeFileSync.json

```diff
@@ -0,0 +1,108 @@
+{
+  "app-id": "org.freefilesync.FreeFileSync",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "FreeFileSync",
+  "finish-args": [
+    "--share=network",
+    "--socket=x11",
```

Comment:
needs --share=ipc as well if you have X11

Reply:
Worked for me even on X11, *shrug*. Added.

Reply:
Yeah, it works - but it's slower. From [the docs](http://docs.flatpak.org/en/latest/working-with-the-sandbox.html):

> --share=ipc means that the sandbox shares IPC namespace with the host. This is not necessarily required, but without it the X shared memory extension will not work, which is very bad for X performance.